src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -273,12 +278,21 @@ def load(
 
             missing_secure_props.extend(profile_loaded.missing_secure_props)
 
-            if override_with_env:
-                env_var = {**self.get_env(cfg)}
 
             if i == 1 and profile_props:
                 break  # Skip loading from global config if profile was found in project config
 
+        usrProject = self.project_user_config.profiles
+        project = self.project_config.profiles
+        prjt = always_merger.merge(usrProject, project)
+
+        usrGlobal = self.global_user_config.profiles
+        glbal = self.global_config.profiles
+        glbl = always_merger.merge(usrGlobal, glbal)
```

Comment:
The `merge` function modifies the contents of the first object passed in. We don't want to be modifying the `profiles` property on config classes - it should match what was loaded from the zowe.config.json file.

I think we should use the `deepcopy` method from the Python `copy` module to create copies of `usrProject` and `usrGlobal` before performing the merge.

Reply:
I tried using the `deepcopy` like this `usrGlobal = deepcopy(self.global_user_config.profiles)` but it doesn't work